corehq/apps/fixtures/upload/workbook.py

```diff
@@ -26,7 +26,7 @@ class _FixtureWorkbook(object):
     def __init__(self, file_or_filename):
         try:
             self.workbook = excel_get_workbook(file_or_filename)
-        except WorkbookJSONError:
+        except WorkbookJSONError as e:
             raise FixtureUploadError([six.text_type(e)])
```

Comment:
I think you want to change this to `FAILURE_MESSAGES['not_excel_file']` or change the failure message that's expected in the test. There was a different error message when I was testing on my dev branch earlier

Reply:
c63c22ccc58